releasenotes/notes/profiler-3-11-traceback-fix-4c98c3ce96833df6.yaml

```
@@ -0,0 +1,5 @@
+---
+fixes:
+  - |
+    profiling: This fix resolves an issue in where the stack collector did not properly access PyFrameObject
+    member values, as PyFrameObject is now created and computed lazily in Python 3.11.
```

Reading this release note it's unclear how this issue manifests. Are we fixing segfaults, memory leaks, improving the accuracy of the profiler, etc....?

```suggestion
    profiling: This fix resolves an issue in where the stack collector did not properly access PyFrameObject
    member values, as PyFrameObject is now created and computed lazily in Python 3.11.

```

---

This is a compatibility issue with Python 3.11.